refactor(packages/sui-logger): rename Reporter and migrate to SWC (#1841)
SUI-Components/sui: packages/sui-logger/README.md

- **`userId` {String}** - User id to add to all logs
- **`trackerName` {String}** [optional] - Tracker name that will be used by the microservice

>   > 💡 Please note that, if you decide to use a custom trackerName, you'd need to create a custom provider inside the ms-adit--reporter microservice.

Reviewer: I guess we should write a text more generic here...

```suggestion
  > 💡 Please note that, if you decide to use a custom trackerName, you'd need to create a custom provider inside your service.
```

Or even we could omit this sentence 😅
